src/state/types.ts

```diff
@@ -153,15 +153,106 @@ export interface CollectiblesState {
   }
 }
 
+// Predictions
+
+export enum BetPosition {
+  BULL = 'Bull',
+  BEAR = 'Bear',
+  HOUSE = 'House',
+}
+
+export enum PredictionStatus {
+  INITIAL = 'initial',
+  LIVE = 'live',
+  PAUSED = 'paused',
+  ERROR = 'error',
+}
+
+export interface Round {
+  id: string
+  epoch: number
+  failed?: boolean
+  startBlock: number
+  startAt: number
+  lockAt: number | null
```

Comment:
Do you specifically want to avoid the `undefined` case ? because `lockAt?: number` is a better solution otherwise